### PR TITLE
Custom scrollbar

### LINKED ISSUE #121: Add a beautiful scroll bar
Add a beautiful scroll bar

### PR DESCRIPTION
**Pull request by william-monroy**
## Purpose
Add custom scrollbar for landing page.
The purpose of this Pull Request is to fix #121 

## Goals
Improve the look of the website.

## Approach
Using `::webkit-scrollbar` attribute of `CSS`

### Screenshots
custom-scrollbar

### Preview Link

https://pr-125-NEW-YOSHITHA-WEBSITE.surge.sh

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: dark/index.html [hunk 1]
```diff
@@ -63,11 +70,86 @@
...
+      type="image/png"
+      href="assets/img/Yoshitha Rathnayake 2.png"
+      alt="Yoshitha Rathnayake"
+    />
 
     <!-- Title -->
 
     <title>YOSHITHA RATHNAYAKE</title>
+
+    <style>
```
Comment (YoshithaRathnayake) [inline dark/index.html:84]: Great Job! @william-monroy but some changes are requested, Can you put these internel css to the externel css file assets/css/styles.css

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html (hunk 1 above)
Comment (william-monroy) [reply dark/index.html]: Sure! :octocat:

Comment (YoshithaRathnayake) [reply dark/index.html]: Can you please remove the styles in the light/index.html file also @william-monroy

**Review by YoshithaRathnayake — APPROVED** [review]
Good Job !
Thank You @william-monroy

**Pull request merged**